### ISSUE TITLE
[TBD] Skeleton loading for gateway for smoother look & feel

### ISSUE THREAD

**Issue by wsdt** [bobanetwork/boba_legacy#1067]
Please check with @InoMurko if it's worth doing or not. 

## Issue Type

```
[ ] bug report
[ x] feature request
```

## Current Behavior
Skeleton loading is perceived as much faster and better by users than a generic loader & just a blank page. 

## Expected Behavior
Consider using a plug&play lib like (just an idea): 
https://www.npmjs.com/package/react-loading-skeleton
Gives a much smoother look & feel.

**Comment by sk-enya** [comment]
Let's explore this if it can help us to solve the cacheing issues on client side.
https://tanstack.com/query/latest/docs/react/overview

**Comment by InoMurko** [comment]
what is [TBD]?

**Comment by wsdt** [comment]
> what is [TBD]?

To-Be-Discussed, since that's just a suggestion from my side - but finally it's up to you or the team if we want to implement something like this.

**Comment by sk-enya** [comment]
We've made several code modifications, including implementing lazy loading, gzipping the bundle, reducing bundle size, and eliminating redundant calls. Additionally, we've relocated images to a separate repository, which was previously causing longer website load times. As a result, the website now appears significantly faster, and I no longer see the need to include a skeleton loader.

@wsdt However, I'm open to hearing your thoughts! 
Closing for now we can reopen again in new repo.
